Dshop/apps/users/api_urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .api_views import RegistrationView
+
+urlpatterns = [
+    path('registration/', RegistrationView.as_view(), name='api-registration'),
```

Comment:
Let's follow REST notation: this one should be /users/ as for create (POST) request. 
Actually, the interesting thing here is: should this not be viewset? Simple
- create - registration
- list - disabled
- update /users/{id} - user can edit his profile.

Food for thought, as we have relation in user so it might not be best approach.

Reply:
this url is included in general urls as /api/users/

Reply:
my feeling is that register should be standalone endpoint